test/e2e/pages/newFolderFlow.ts

```diff
@@ -53,13 +53,24 @@ export class NewFolderFlow {
 	 * @param initGitRepo Whether to initialize a Git repository.
 	 **/
 	async setFolderNameLocation(options: CreateFolderOptions) {
-		const { folderName, initGitRepo } = options;
+		const { folderName, initGitRepo, createPyprojectToml, folderTemplate: type } = options;
 
 		await this.folderNameInput.fill(folderName);
 		if (initGitRepo) {
 			await this.code.driver.page.getByText('Initialize Git repository').check();
 		}
 
+		if (type === FolderTemplate.PYTHON_PROJECT) {
+			const checkboxLabel = this.code.driver.page.getByText('Create pyproject.toml file');
+			const shouldBeChecked = createPyprojectToml ?? false;
+			if (!shouldBeChecked) {
+				// It's checked by default, so click to uncheck
+				await checkboxLabel.click();
```

Comment:
I don't know why the `.check()` and `.uncheck()` methods weren't working on this element when they seem to be working on the Git checkbox above. Oh well, this works.